kDriveCore/Data/Cache/DriveFileManager/DriveFileManager.swift

```diff
@@ -187,7 +189,14 @@ public final class DriveFileManager: Transactionable {
             },
             objectTypes: DriveFileManager.constants.driveObjectTypes
         )
+    }
+
+    init(drive: Drive, apiFetcher: DriveApiFetcher, context: DriveFileManagerContext = .drive) {
+        self.drive = drive
+        self.apiFetcher = apiFetcher
+        realmConfiguration = Self.configuration(context: context, driveId: drive.id, driveUserId: drive.userId)
 
+        let realmURL = context.realmURL(driveId: drive.id, driveUserId: drive.userId)
```

Comment:
Since the DriveFileManager can be dynamically spawn for _any_ drive it does not make sense to put it in DI.